ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/EthSigner.java

```diff
@@ -49,6 +49,7 @@ public EthSigner(final Config config, final TransactionSignerProvider transactio
   }
 
   public void run() {
+    LOG.info("### Starting run");
```

Comment:
are these meant to be removed or lower to trace?

Reply:
removed - oops.